CHANGELOG.md

```diff
@@ -3,8 +3,11 @@
 ## v0.0.17 (unreleased)
 
 - Fix locale warnings on MacOS
+
 - Make `executables` a list, and optional, in `mkHaskellPackage`
+
```

Comment:
I've been meaning to ask - what's the deal with spaces in the changelog? In the unreleased version we have it, but not in the others? Why not be consistent throughout?

Reply:
I'm not sure. This was my attempt at just making in consistent within this section, without having to go back to older sections and change all of them. But yeah, I think I'll just remove all newlines like this.

Reply:
Done in 8ab84b72.